lib/mayaUsd/render/vp2RenderDelegate/mesh.cpp

```diff
@@ -1333,13 +1337,34 @@ void HdVP2Mesh::_InitRepr(const TfToken& reprToken, HdDirtyBits* dirtyBits)
         case HdMeshGeomStyleHull:
             // Creating the smoothHull hull render items requires geom subsets from the topology,
             // and we can't access that here.
-#ifdef HAS_DEFAULT_MATERIAL_SUPPORT_API
+#ifdef MAYA_NEW_POINT_SNAPPING_SUPPORT
             if (reprToken == HdVP2ReprTokens->defaultMaterial) {
                 // But default material mode does not use geom subsets, so we create the render item
                 renderItem = _CreateSmoothHullRenderItem(renderItemName);
                 renderItem->setDefaultMaterialHandling(
                     MRenderItem::DrawOnlyWhenDefaultMaterialActive);
                 renderItem->setShader(_delegate->Get3dDefaultMaterialShader());
+
+                if (!GetInstancerId().IsEmpty()) {
+                    MString shadedSelectedInstancesRenderItemName = renderItemName;
+                    shadedSelectedInstancesRenderItemName
+                        += std::string(1, VP2_RENDER_DELEGATE_SEPARATOR).c_str();
+                    shadedSelectedInstancesRenderItemName += "shadedSelectedInstances";
```

Comment:
Need one definition for the "shadedSelectedInstances" string which appears 3 times. Maybe a token? Or maybe this code should be in a helper function?